ci/infra/testrunner/platforms/libvirt.py

```diff
@@ -16,3 +16,19 @@ def _env_setup_cmd(self):
     @timeout(600)
     def _cleanup_platform(self):
         self.destroy()
+
+    def _update_platform_tfvars(self, tfvars):
+        """Update the tfvars that are specific to the platform"""
+        new_vars = {
+            "libvirt_uri": self.conf.libvirt.uri,
+            "libvirt_keyfile": self.conf.libvirt.keyfile
+        }
+
+        for k, v in new_vars.items():
```

Comment:
should this be the same in a generic method and the new_vars be a class attribute? That way, less code :D